build-system/tasks/presubmit-checks.js

```diff
@@ -257,6 +257,7 @@ var forbiddenTerms = {
     whitelist: [
       'src/service/viewer-impl.js',
       'src/service/storage-impl.js',
+      'src/service/performance-impl.js',
```

Comment:
@erwinmombay I saw we have whitelist rule for `sendMessage` but not the `sendMessageCancelUnsent` (previously `sendMessageUnreliable`). Do we need to add rules for `sendMessageCancelUnsent` too?